openstates/ar/legislators.py

```diff
@@ -48,7 +53,13 @@ def scrape_member(self, chamber, term, member_url):
 
         party = name_and_party[-1]
 
-        if party == '(R)':
+        if ' '.join(name_and_party[1:]) == 'David Johnson' and "(" not in party:
+            full_name = ' '.join(name_and_party[1:])
+            party = 'Democratic'
+            global _Remove_special_casing
```

Comment:
You don't need to declare a `global` variable; it already exists since you declared it on line 8.